src/cmds/qstat.c

```diff
@@ -545,6 +549,32 @@ prt_attr(char *name, char *resource, char *value, int one_line) {
 #define SIZETSK 5	/* length of tsk in printf */
 #define SIZEUSER 15	/* length of user name in printf */
 
+
+/**
+ * @brief
+ *	Check if value is too long, truncate and append with * if needed
```

Comment:
Don't say '*'.  It's dependent on what your macro is set to.  I'd say 'and indicate it is truncated' or something like that.